### PR TITLE
 changes/in the login register.

### PR DESCRIPTION
**Pull request by juliettesmc**
new components and changes in the login register.

### CODE REVIEW

**Comment by AmyJenSandoval** [comment]
Instead of using a "Next" route, use useState hook to move between the different states of the multi-step form. Search "multi-step form React" for examples.

**Comment by AmyJenSandoval** [comment]
https://github.com/Code-the-Dream-School/cc-prac-team3-front/assets/98506565/930e9d22-ca75-4e25-b05d-a38b51189f93


Can you separate the Password and Confirm Password? When the user types in their password, it is filling out both sections, but it should be separate so that the user can type in their password two times.

**Review by leo-mutuku — COMMENTED** [review]
This is okay, component state variables well defined, react routes well implimented

File: src/components/Navbar.jsx
Comment (AmyJenSandoval) [inline src/components/Navbar.jsx:1]: Please delete Navbar component. Melise completed navigation menu.

File: src/components/Register.jsx [hunk 1]
```diff
@@ -1,28 +1,58 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import "./App.css"
 
 export const Register = () => {
+    const navigate = useNavigate();
+
     const [email,setEmail]=useState("");
     const [pass,setPass]=useState("");
     const [Name,setName]=useState("");
```
Comment (AmyJenSandoval) [inline src/components/Register.jsx]: Name should be lowercase "name"

**Comment by AmyJenSandoval** [comment]
Git pull dev, then checkout 'changes-register' and merge dev into 'changes-register'. Then, run 'npm i' to install new icon packages. Thank you!

File: src/index.js [hunk 2]
```diff
@@ -16,6 +21,10 @@ root.render(
       <Route exact path='/Login'element={<Login/>}/> 
       <Route exact path='/register'element={<Register/>}/> 
       <Route exact path='/ForgotPassword'element={<ForgotPassword/>}/> 
+      <Route exact path='/ResetPassword'element={<ResetPassword/>}/> 
+      <Route exact path='/'element={<Cart/>}/>
```
Comment (AmyJenSandoval) [inline src/index.js]: Please delete line 25 in index.js
<Route exact path='/'element={<Cart/>}/>

"/" will be used for the home path

**Pull request merged**